### PR TITLE
Add manual page about updating Jenkins

### PR DESCRIPTION
**Pull request by suzannehamilton**
It would be good to automate these processes, but we can't easily update Jenkins from within Jenkins, so these steps may be manual unless we think it's worth spending the time automating them.

### CODE REVIEW

File: manual/update-jenkins.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Jenkins needs to be kept up to date in several ways: we need to update the EC2
+instance it runs on, the Jenkins application itself, and all of its plugins.
+
+All the code needed to update Jenkins is in the [tdr-jenkins] repo.
+
+## Preparation
+
+**Before running any updates**:
+
+* Run the [TDR Jenkins Backup]
```
Comment (TomJKing) [inline manual/update-jenkins.md:12]: This link goes to the Jenkins git repo. Is that correct, as there doesn't appear to be any instructions here on running the backup.

File: manual/update-jenkins.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+* Warn other developers that Jenkins will be unavailable for a few minutes
+
+It is a good idea to run the [TDR Jenkins Backup] job before deploying Jenkins.
```
Comment (TomJKing) [inline manual/update-jenkins.md:15]: See previous comment about this link.

File: manual/update-jenkins.md (hunk 1 above)
Comment (suzannehamilton) [reply manual/update-jenkins.md:12]: Citrix copy-paste fail. Fixed, thanks.

**Comment by suzannehamilton** [comment]
@TomJKing Are you happy with this PR now?

**Pull request merged**